Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayIntegrationTests.swift

```diff
@@ -308,13 +308,20 @@ class SentrySessionReplayIntegrationTests: XCTestCase {
     }
     
     func createLastSessionReplay(writeSessionInfo: Bool = true, errorSampleRate: Double = 1) throws {
-        let replayFolder = SentryDependencyContainer.sharedInstance().fileManager.sentryPath + "/replay"
-        let jsonPath = replayFolder + "/lastreplay"
+        let options = Options()
+        options.dsn = "https://user@test.com/test"
+        options.cacheDirectoryPath = FileManager.default.temporaryDirectory.path
```

Comment:
Doesn't have to hold up this pr, but consider that using a wrapper for this in SentryFileManager and using a predictable, persistent location would allow for better debugging, especially from CI

Reply:
I tried that, SentryFileManager has a weird initialisation, it was easier to hard coded it, which is also predictable I guess.